src/backup/tasks/backup.task.ts

```
@@ -0,0 +1,122 @@
+  async handleDailyBackup() {
+    this.logger.log('Starting daily backup...');
+
+    try {
+      const database = this.configService.get<string>('DB_NAME');
+      if (!database) {
+        throw new Error('DB_NAME environment variable is not set');
+      }
+
+      // Validate AWS configuration for cross-region backups
+      const awsAccessKey = this.configService.get<string>('AWS_ACCESS_KEY_ID');
+      const awsSecretKey = this.configService.get<string>('AWS_SECRET_ACCESS_KEY');
```

ESLint

[error] Replace `'AWS_SECRET_ACCESS_KEY'` with `⏎········'AWS_SECRET_ACCESS_KEY',⏎······`

(prettier/prettier)